openapi.yaml

```diff
@@ -649,3 +649,16 @@ components:
                 format: uuid
                 example: '028e12d1-be7a-4913-b797-7e848c4e4977'
             description: GUID from ERP
+            
+  securitySchemes:
+    oAuth2:
+      type: oauth2
+      description: This API uses OAuth 2 with the implicit grant flow. [More info](https://api.example.com/docs/auth)
```

Comment:
I am not sure where this API will be executed, but I would like to point out, that the OAuth2.0 `implict flow` is nowadays considered as deprecated (due to security concerns) in favor of `Implicit flow + PKCE`

Reply:
this doesn't depend on us, but on implementation of the other side